src/react-components/src/components/01-atoms/Button/Button.tsx

```diff
@@ -42,11 +44,14 @@ export default class Button extends React.Component<ButtonProps, {}> {
       buttonModifiers.push(type);
     }
 
-    let btnContent = [content];
+    let btnContent: any[] = [];
```

Comment:
I'm not a fan of `any`. You can create a type and assign it for the `content` prop:
```
type BtnContent = string | JSX.Element;
...

content: BtnContent; // instead of `content: string | JSX.Element`
...

let btnContent: BtnContent[] = []; // instead of `let btnContent: any[] = [];`
```